app/views/user_mailer/challenge_assignment_notification.text.erb

```diff
@@ -1,9 +1,9 @@
 <% content_for :message do %>
-<%= t ".assignment.text", collection_title: @collection.title, collection_url: collection_url(@collection) %>
+<%= t(".assignment.text", collection_title: @collection.title, collection_url: collection_url(@collection)) %>
 
-<%= t ".recipient" %> <%= @request.nil? ? t(".recipient_missing") : text_pseud(@request.pseud) %>
+<%= metadata_label(t(".recipient")) %><%= @request.nil? ? t(".recipient_missing") : text_pseud(@request.pseud) %>
 
-<%= t ".prompts" %>
+<%= t(".prompts") %>
```

Comment:
Do we need to use `style_metadata_label` for `t(".prompts")`?

Reply:
That's a good question. I left it out because it's not really a metadata label, more of a headline. Because this is on its own line there is no worry with whitespace after the colon. But it could still be switched for consistency, I don't have a strong opinion here.

Reply:
It should be fine for now. We can change it later if necessary.